hazelcast/near_cache.py

```diff
@@ -84,6 +84,7 @@ def __init__(
         eviction_sampling_count=None,
         eviction_sampling_pool_size=None,
     ):
+        super(NearCache, self).__init__()
```

Comment:
Why you added this? I guess you forget to add this before

Reply:
BTW, we may not need extending from a dict. Where do we need dict extension? I think we can only implement __getitem__ and __setitem__ to getter and setter behaviour with [].

Reply:
1. Yes
2. It was like this before and I don't see a point in changing it.